plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -804,16 +804,21 @@ void GSRendererHW::SwSpriteRender()
 
 	bool alpha_blending_enabled = PRIM->ABE;
 
-	GSVector4i vc = m_vt.m_min.c;  // 0x000000AA000000BB000000GG000000RR
-	vc = vc.ps32();                // 0x00AA00BB00GG00RR00AA00BB00GG00RR
+	// TODO Is last vertex m_vertex.buff[m_vertex.tail - 1] or m_vertex.buff[m_vertex.next - 1]?
+	GSVertex& v = m_vertex.buff[m_vertex.tail - 1];  // Last vertex.
+	GSVector4i vc = GSVector4i(v.RGBAQ.R, v.RGBAQ.G, v.RGBAQ.B, v.RGBAQ.A);  // 0x000000AA000000BB000000GG000000RR
+	vc = vc.ps32();  // 0x00AA00BB00GG00RR00AA00BB00GG00RR
 
 	GSVector4i a_mask = GSVector4i::xff000000().u8to16();  // 0x00FF00000000000000FF000000000000
 
 	bool fb_mask_enabled = m_context->FRAME.FBMSK != 0x0;
 	GSVector4i fb_mask = GSVector4i(m_context->FRAME.FBMSK).xxxx(); // 0x????????????????MMMMMMMMMMMMMMMM
 
+	uint8 tex0_tfx = m_context->TEX0.TFX;
```

Comment:
Maybe use constant variables when applicable throughout the code, just makes the code cleaner and less VS warnings to deal with.